### DISCUSSION TITLE
请问如何实现群禁言？

### DISCUSSION THREAD

**Discussion by superboyyy** [takayama-lily/oicq#444] [Q&A]
如题，现在已经可以实现检测到群聊关键词撤回消息，禁言应该用什么方法？

**Comment by dogxii** [comment]
Client.setGroupBan(group_id, user_id, duration)
参数 | 类型 | 默认 | 说明
-- | -- | -- | --
group_id | number |   | 群号
user_id | number |   | qq
duration? | number | 1800 | 禁言时间（秒）
